docs/source/common_issues.rst

```diff
@@ -262,7 +269,8 @@ explicit type cast:
    def f(o: object) -> None:
        if type(o) is int:
            o = cast(int, o)
-           g(o + 1)    # This would be an error without the cast
+           g(o + 1)  # This would be an error without the cast:
+                     # error: Unsupported operand types for + ("object" and "int")
```

Comment:
I think it's confusing to write out the error since it doesn't actually happen. I would just leave this comment as is.

Alternatively, you can have the following:
```
g(o + 1)  # error: Unsupported operand types for + ("object" and "int")
o = cast(int, o)
g(o + 1)  # This would be an error without the cast
```